controllers/apply/forms/reaching-communities-form.js

```diff
@@ -100,70 +100,92 @@ formModel.registerStep({
                             }
                         });
                     }
+                },
+                {
+                    type: 'text',
+                    name: 'project-location',
+                    label: "In your own words, describe the location(s) that you'll be running your project(s) in",
+                    placeholder: 'eg. "Newtown community centre" or "Alfreton, Derby and Ripley"',
+                    isRequired: true,
+                    size: 60,
+                    validator: function(field) {
+                        return check(field.name)
+                            .trim()
+                            .not()
+                            .isEmpty()
+                            .withMessage('Project location must be provided');
+                    }
                 }
             ]
         }
     ]
 });
 
 formModel.registerStep({
-    name: 'Your Organisation',
+    name: 'Your organisation',
     fieldsets: [
         {
-            legend: 'Your Organisation',
+            legend: 'Your organisation',
             fields: [
                 {
                     type: 'text',
                     name: 'organisation-name',
-                    label: 'Legal Name',
+                    label: 'Legal name',
                     isRequired: true,
-                    canBeDuplicated: true,
-                    duplicateLabel: 'Add another organisation',
-                    duplicateHelpText:
-                        'If you’re working with other organisations to deliver your idea, list them below. If you don’t know yet we can discuss this later on.',
                     validator: function(field) {
                         return check(field.name)
                             .trim()
                             .not()
                             .isEmpty()
                             .withMessage('Organisation must be provided');
                     }
+                },
+                {
+                    type: 'text',
```

Comment:
Does this do anything yet?

Reply:
How do you mean? It outputs an `<input type="text" />` field...?